src/components/Organization/pages/Passwords.js

```diff
@@ -42,24 +71,24 @@ function Passwords() {
         data-testid="organization-passwords-page"
       >
         <Grid item xs={12}>
-          <Typography className={classes.heading}>Passwords</Typography>
+          <Typography className={classes.responsiveHeading}>Passwords</Typography>
         </Grid>
-        <Grid item xs={12}>
+        <Grid className={classes.responsiveInput} item xs={12}>
           <Typography>Old Password</Typography>
           <InputBase className={classes.input} placeholder="Old Password" />
         </Grid>
-        <Grid item xs={12}>
+        <Grid className={classes.responsiveInput} item xs={12}>
```

Comment:
className `responsiveInput`  or `responsiveHeading` is kind of verbose. 

please update it to just `inputContainer` and `heading`